app.js

```diff
@@ -262,6 +266,12 @@ function tick() {
   if (liveCells === 0) {
     const stopButton = document.querySelector('#stop');
     stopButton.click();
+
+    //this doesn't work but lines 284 seem out of place. 
```

Comment:
'this doesn;t work' ie the lines I added then commented out ... bc of ticker reference, or lack therof